src/modules/clusters/cluster-list/cluster-list.js

```diff
@@ -364,6 +361,26 @@
                 cluster.isProfilingEnabled === "Unknown";
         }
 
+        function getTemplate(cluster) {
+            var template = "";
+
+            if (cluster.currentStatus === "failed") {
+
+                if (cluster.jobType === "UnmanageCluster") {
+                    template = "If unmanage fails, resolve the issue and reinitiate unmanage cluster.";
+                } else if (cluster.jobType === "ImportCluster") {
+                    template = "If import cluster fails, resolve the issue before performing an unmanage cluster and reinitiate import.";
+                } else if (cluster.jobType === "ExpandClusterWithDetectedPeers") {
+                    template = "If cluster expansion fails, check if tendrl-ansible was executed successfully and ensure the node agents are correctly configured.";
+                }
+
+            } else if (cluster.managed === "Yes" && cluster.currentStatus !== "failed" && cluster.currentStatus !== "in_progress") {
```

Comment:
Why are we checking the opposite value(cluster.currentStatus !== "failed" && cluster.currentStatus !== "in_progress"). I hope there will be status that days "done" on "success".

can we check for that?